crates/config/src/pathfinder.rs

```diff
@@ -0,0 +1,105 @@
+use clap::ValueEnum;
+use madara_cli_common::{Prompt, PromptSelect};
+use strum::{EnumIter, IntoEnumIterator};
+
+const DEFAULT_CHAIN_ID: &str = "MADARA_DEVNET";
+const DEFAULT_GATEWAY_URL: &str = "http://madara:8080/gateway";
+const DEFAULT_FEEDER_GATEWAY_URL: &str = "http://madara:8080/feeder_gateway";
+const DEFAULT_HTTP_RPC: &str = "0.0.0.0:9545";
+const DEFAULT_DATA_DIRECTORY: &str = "/usr/share/pathfinder/data";
+
+#[derive(Debug, Clone, Copy, ValueEnum, PartialEq, Eq, Default, EnumIter, strum::Display)]
+pub enum PathfinderNetwork {
+    #[default]
+    Custom,
+    Sepolia,
+    Mainnet,
+}
+
+pub struct PathfinderRunnerConfigMode {
+    pub network: Option<PathfinderNetwork>,
+    pub chain_id: Option<String>,
+    pub ethereum_url_wss: Option<String>,
+    pub gateway_url: Option<String>,
+    pub feeder_gateway_url: Option<String>,
+    pub http_rpc: Option<String>,
+    pub data_directory: Option<String>,
+}
+
+impl Default for PathfinderRunnerConfigMode {
+    fn default() -> Self {
+        Self {
+            network: Some(PathfinderNetwork::Custom),
+            chain_id: Some(DEFAULT_CHAIN_ID.to_string()),
+            ethereum_url_wss: Some("RPC_API_KEY".to_string()),
+            gateway_url: Some(DEFAULT_GATEWAY_URL.to_string()),
+            feeder_gateway_url: Some(DEFAULT_FEEDER_GATEWAY_URL.to_string()),
+            http_rpc: Some(DEFAULT_HTTP_RPC.to_string()),
+            data_directory: Some(DEFAULT_DATA_DIRECTORY.to_string()),
+        }
+    }
+}
+
+impl PathfinderRunnerConfigMode {
+    pub fn fill_values_with_prompt(self) -> anyhow::Result<PathfinderRunnerConfigMode> {
+        let network = self.network.unwrap_or_else(|| {
+            PromptSelect::new("Select network", PathfinderNetwork::iter()).ask()
+        });
+
+        let chain_id = self.chain_id.unwrap_or_else(|| {
+            Prompt::new("Input chain-id")
+                .default(DEFAULT_CHAIN_ID)
+                .ask()
+        });
```

Comment:
Is there a flow that ask for this inputs?

Reply:
Not yet, but the params were added to the struct. It's highly probable that the user wants to set a different chain id for different networks.
This would be one (of many) placeholders that must be improved under a customization feature.